tests/forms_tests/field_tests/test_filepathfield.py

```diff
@@ -18,80 +18,79 @@ def fix_os_paths(x):
 class FilePathFieldTest(SimpleTestCase):
 
     def test_filepathfield_1(self):
-        path = os.path.dirname(os.path.abspath(forms.__file__)) + '/'
-        self.assertTrue(fix_os_paths(path).endswith('/django/forms/'))
+        path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'filepathfield_test_dir') + '/'
+        self.assertTrue(fix_os_paths(path).endswith('/filepathfield_test_dir/'))
 
     def test_filepathfield_2(self):
-        path = os.path.dirname(os.path.abspath(forms.__file__)) + '/'
+        path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'filepathfield_test_dir') + '/'
         f = FilePathField(path=path)
-        f.choices = [p for p in f.choices if p[0].endswith('.py')]
-        f.choices.sort()
         expected = [
```

Comment:
Perhaps the full list could be a class attribute so it doesn't have to be repeated several times.